modules/cross-account-member/variables.tf

```diff
@@ -13,12 +13,24 @@ variable "action_targets" {
   default = []
 }
 
+variable "auto_enable_controls" {
+  description = "Boolean that enables the security standards that Security Hub has designated as automatically enabled including: `AWS Foundational Security Best Practices v1.0.0` and `CIS AWS Foundations Benchmark v1.2.0`"
+  type        = bool
+  default     = true
+}
+
 variable "control_finding_generator" {
   description = "(Optional) Updates whether the calling account has consolidated control findings turned on."
```

Comment:
Update the description here to match the others?

Reply:
done